Create ui-c8y-1023-4-6-info-tab-should-be-editable-when-dashboard-manager-feature-preview-is.md (#3955)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-4-6-info-tab-should-be-editable-when-dashboard-manager-feature-preview-is.md

---
date: ""

> title: Info tab was hidden when Dashboard manager feature preview is disabled

Reviewer: ```suggestion
title: Info tab no longer hidden when dashboard manager feature preview is disabled
```

> title: Info tab should be editable when Dashboard manager feature preview is disabled

Reviewer 2: ```suggestion
title: Info tab was hidden when Dashboard manager feature preview is disabled
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5178
version: 1023.4.6
---

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer 2: ```suggestion
With the rollout of the **Dashboard manager** feature preview the **Info** tab was hidden for some devices in **Device management** application. This issue is now fixed and **Info** tab will be visible again for all devices when feature preview is disabled. For devices for which the **Info** tab was still visible the **Edit widgets** and **Dashboard settings** options were disabled. This is now also fixed.
```

> With the rollout of the **Dashboard manager** feature preview the **Info** tab was hidden for some devices in **Device management** application. This issue is now fixed and **Info** tab will be visible again for all devices when feature preview is disabled. For devices for which the **Info** tab was still visible the **Edit widgets** and **Dashboard settings** options were disabled. This is now also fixed.

Reviewer: ```suggestion
With the rollout of the **Dashboard manager** feature preview in the Device Management application, the **Info** tab was hidden for some devices when the feature preview was disabled. This issue is now fixed, and the **Info** tab is visible again for all devices when the feature preview is disabled. Moreover, issues with the **Edit widgets** and **Dashboard settings** options in the same context have been fixed. 
```
